docs/UserGuide.md

```diff
@@ -669,52 +679,52 @@ worries! You can edit the details of a customer simply by calling this command.
 
 **Examples:**
 
-- `customer edit 1 --name Gabriel -–phone 9813 1051`{.swift} </br>
-  Edits customer 1's name to Gabriel and phone to 9813 1051.
-- `customer edit 2 --name Joe --email yangyang@gmail.com -–address Block 10 Tampines Road`{.swift} </br>
-  Edits customer 2's name to Joe, email to yangyang@gmail.com and address to Block 10 Tampines Road.\
+- `customer edit 1 --name Gabriel --phone 9813 1051`{.swift} </br>
+  Edits Customer 1's name to Gabriel and phone to 9813 1051.
+- `customer edit 2 --name Joe --email yangyang@gmail.com --address Block 10 Tampines Road`{.swift} </br>
+  Edits Customer 2's name to Joe, email to yangyang@gmail.com and address to Block 10 Tampines Road.\
 
   [&uarr; Back to Table of Contents](#top)
 
 ### Delete a Customer
 
-> Deletes the specified customer from the customer database. All deliveries associated with the customer will also be
+> Deletes the specified Customer from the Customer database. All Deliveries associated with the Customer will also be
 > deleted.
 
-_You can delete any customers who no longer require your services from HomeBoss by calling this command._
+_You can delete any Customers who no longer require your services from HomeBoss by calling this command._
 
 **Format:** `customer delete CUSTOMER_ID`{.swift}
 
-<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
-
-**Note:**
+<box background-color="#f2dede" border-color="#ebccd1" type="warning" icon=":exclamation:">
 
-* `CUSTOMER_ID`{.swift} must be an integer greater than 0 that corresponds to an existing Customer.
+**Warning:**
 
+Be careful! You won't be able to undo this edit action!
 </box>
 
-<box background-color="#f2dede" border-color="#ebccd1" type="warning" icon=":exclamation:">
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
 
-**Warning:**
+**Note:**
+
+* `CUSTOMER_ID`{.swift} must be an integer greater than 0 that corresponds to an existing Customer.
 
-Be careful! You won't be able to undo this edit action!
 </box>
 
 **Example:**
 
 * `customer delete 1`{.swift} </br>
-  Deletes the customer with ID 1.
+  Deletes the Customer with ID 1.
 
 [&uarr; Back to Table of Contents](#table-of-contents)
 
 ## <span style="text-decoration:underline;">Delivery</span>
 
 ### Add a Delivery
 
-> Adds a delivery to the delivery book.
+> Adds a Delivery to the delivery book.
```

Comment:
Actually, we're removing this line and merge them together with the description below right?

Reply:
yep